catalogs/catalog-common/src/main/java/org/apache/gravitino/credential/CredentialConstants.java

```diff
@@ -22,6 +22,8 @@
 public class CredentialConstants {
   public static final String CREDENTIAL_PROVIDER_TYPE = "credential-provider-type";
   public static final String CREDENTIAL_PROVIDERS = "credential-providers";
+  public static final String CREDENTIAL_CACHE_EXPIRE_IN_SECS = "credential-cache-expire-in-secs";
```

Comment:
I feel that this property may not so useful, you can refer to Spark's token refresh config, set a ratio when to refresh the token.

Reply:
I don't think we should refresh tokens periodic, the ideal implement may prefetch credentials according to the access rate of the credential, but this make cache complicated, I prefer to optimize it latter.

Reply:
use `credential-cache-expire-ratio`